docs/examples/blocking_consume_recover_retry.rst

```diff
@@ -0,0 +1,44 @@
+Using the Blocking Connection with connection recovery decorator
+================================================================
+
+.. _example_blocking_basic_consume_recover_retry:
+
+Connection recovery is mostly valuable in case of consumers, because they are
+long-running tasks doing smaller pieces of work on each message, hence they
+can be restarted on errors.
+
+For BlockingConnection adapter exception handling can be used to check for
+connection errors.
+
+You can use decorators from libraries like `retry` to set up recovery behaviour.
```

Comment:
Perhaps add to the description an example of installing the retry decorator. Is it `pip install retry`?